app/views/shared/_feed.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= link_to "Subscribe", root_path, class: "btn btn-info btn-xs" %>
+<div class="row">
+  <% if posts.nil? || posts.empty? %>
+    <h2>This feed doesn't have any posts. ;_;</h2>
+  <% else %>
```

Comment:
Someone had mentioned this on the last project but I wasn't sure how to implement it. This makes a lot of sense, thank you!